packages/ui/src/ui/pages/components/tabs/nodes/Nodes/NodesColumnHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {COMPONENTS_NODES_TABLE_ID} from '../../../../../constants/components/nodes/nodes';
+import ColumnHeader, {
+    ColumnHeaderProps,
+    HasSortColumn,
+} from '../../../../../components/ColumnHeader/ColumnHeader';
+import {toggleColumnSortOrder} from '../../../../../store/actions/tables';
+import {getTables} from '../../../../../store/selectors/tables';
+import {NodesTableColumnNames} from '../../../../../utils/components/nodes/tables';
+import {oldSortStateToOrderType} from '../../../../../utils/sort-helpers';
+
+export function NodesColumnHeader(
+    props: HasSortColumn<NodesTableColumnNames> & Pick<ColumnHeaderProps, 'align'>,
+) {
+    const dispatch = useDispatch();
+    const sortState = useSelector(getTables)[COMPONENTS_NODES_TABLE_ID];
+    const order = oldSortStateToOrderType(sortState);
+
+    const column = props.options?.find(({column}) => sortState.field === column);
+
+    return (
+        <ColumnHeader
+            {...props}
+            column={props.options ? column?.column ?? props.column : props.column}
```

Comment:
double ternary operator. they seem so convenient but this is the path to evil

Reply:
not double ternary operator